core/tests/gae_suite.py

```diff
@@ -63,6 +63,7 @@
     os.path.join(THIRD_PARTY_DIR, 'six-1.15.0'),
     os.path.join(THIRD_PARTY_DIR, 'soupsieve-1.9.5'),
     os.path.join(THIRD_PARTY_DIR, 'webencodings-0.5.1'),
+    os.path.join(THIRD_PARTY_DIR, 'redis-3.5.3')
```

Comment:
Why are we using 3.5.3 here and elsewhere? I don't see where this gets installed -- the version of redis-cli is 6.0.6.